packages/clerk-js/webpack.config.js

```diff
@@ -238,12 +238,12 @@ const devConfig = ({ mode, env }) => {
         rules: [svgLoader(), typescriptLoaderDev()],
       },
       plugins: [
-        new ReactRefreshWebpackPlugin({ overlay: { sockHost: 'js.lclclerk.com' } }),
+        new ReactRefreshWebpackPlugin({ overlay: { sockHost: 'http://localhost:4000' } }),
```

Comment:
We should derive this from an environment variable to make sure the existing proxy setup still works in `clerk_go`.